pkgs/by-name/lu/lug-helper/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  stdenvNoCC,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  copyDesktopItems,
+  coreutils,
+  findutils,
+  zenity,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  name = "lug-helper";
+  version = "2.18";
+  src = fetchFromGitHub {
+    owner = "starcitizen-lug";
+    repo = "lug-helper";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-x6o9hNXadlZrww5+a9xZtNfRwxKuTO/O9M9iYvhMIYc=";
+  };
+
+  buildInputs = [
+    coreutils
+    findutils
+    zenity
+  ];
+
+  nativeBuildInputs = [
+    copyDesktopItems
+    makeWrapper
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "lug-helper";
+      exec = "lug-helper";
+      icon = "lug-helper";
```

Comment:
```suggestion
      icon = "lug-logo";
```
Upstream uses `lug-logo` instead of the executable name so we should change this to avoid breaking https://github.com/starcitizen-lug/lug-helper/blob/a27a90dea554427b2e5d551f27341ccdf1c6372b/lug-helper.sh#L159-L160